.github/workflows/Tests.yml

```diff
@@ -53,6 +53,28 @@ jobs:
 
     steps:
 
+      - name: Installing Klayout
+        run: |
+          sudo apt update -qq -y
+          sudo apt install -qq -y \
```

Comment:
`apt-get -f install` should work as long as we use the right command `-v` or `-h` for testing (i.e: it shouldn't try to interact with the ui subsystem).